effectful/handlers/torch.py

```diff
@@ -337,6 +337,11 @@ def _embed_tensor(op, *args, **kwargs):
         return _TensorTerm(op, *args, **kwargs)
 
 
+@defterm.register(torch.Tensor)
```

Comment:
`Iterable` is so broad that we'd probably need to implement many overrides like this. `Generator` is narrower by comparison but still covers the syntax we want.

Reply:
Yep, agreed.